### DISCUSSION TITLE
Licensing issue

### DISCUSSION THREAD

**Discussion by Manjusmnr** [pymupdf/PyMuPDF#1949] [General]
Planning to use the library. Licensed under GNU Affero General Public License v3.0 and GNU General Public License v3.0.
We are not doing any modifications in the source code of the library.
GPL, or our application can be made proprietary.
Can you share more insights on this

**Comment by julian-smith-artifex-com** [comment]
I am not a lawyer so nothing i say can be taken as legal advice, but my understanding is that your application needs to use the GNU Affero General Public License if it uses PyMuPDF.

**Comment by JorjMcKie** [comment]
@Manjusmnr - I think we can close the issue, correct?

**Comment by JorjMcKie** [comment]
@julian-smith-artifex-com - I think we can close this, or may be there is some value in transferring the issue to the Discussions tab ... under a new topic like "licensing"?

**Comment by julian-smith-artifex-com** [comment]
I should have mentioned that PyMuPDF is also available under a commercial license from Artifex.

The full information about licensing is at: https://pymupdf.readthedocs.io/en/latest/intro.html#license-and-copyright

[@JorjMcKie You're right about moving this to Discussions, i'll do this now. And maybe close it soon unless there are more comments.]
